### PR TITLE
allow disabling auto updates

### PR DESCRIPTION
**Pull request by Pl8tinium**

### CODE REVIEW

**Review by Geo25rey — CHANGES_REQUESTED** [review]
I think auto update should be the default and having the ability to opt-out is fine, but I think the risk of being excluded from the network if your node is out of date should be mentioned.

**Comment by Geo25rey** [comment]
You should be able to use the `enabled` flag on the watchtower container combined with the `.env` file accessible in the `docker-compose.yml` https://docs.docker.com/compose/environment-variables/env-file/

**Comment by Pl8tinium** [comment]
https://containrrr.dev/watchtower/container-selection/

aah i missed that 🙂

File: docker-compose.yml [hunk 1]
```diff
@@ -72,15 +76,15 @@ services:
...
           max-size: "5m"
+    labels:
+      - "com.centurylinklabs.watchtower.enable=${AUTO_UPDATE:-true}"
 
   # Watchtower automatically updates the containers
   watchtower:
     image: containrrr/watchtower
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
     command: --interval 3600 --cleanup
```
Comment (Geo25rey) [inline docker-compose.yml]: I thought you were gonna add the `enabled` flag to the `watchtower` Docker Compose service defined in `docker-compose.yml`, but this works too. It's probably a good idea to enable the whitelist command option to watchtower like this, too. 
https://containrrr.dev/watchtower/arguments/#filter_by_enable_label

As long as you tested the `.env` `AUTO_UPDATE` value locally, LGTM

```suggestion
    command: --label-enable --interval 3600 --cleanup
```

**Review by Geo25rey — APPROVED** [review]

**Comment by Pl8tinium** [comment]
image

tested locally, works

**Pull request merged**